ph5/clients/tests/test_ph5availability.py

```diff
@@ -1120,8 +1120,8 @@ def test_main(self):
                 out.compare(
                     "#n s     l  c   q                    earliest"
                     "                      latest\n"
-                    "AA 0407  -- HHN   2018-12-17T22:20:30.917000Z"
-                    " 2018-12-17T22:20:40.922000Z\n"
+                    "AA 0407  -- HHN   2018-12-17T22:20:30.681998Z"
+                    " 2018-12-17T22:20:40.691998Z\n"
```

Comment:
Giving -S option should give the same output as original test.

Reply:
The test suite was reverted back to the original test and now passes.